identity/app/views/consentJourney.scala.html

```diff
@@ -241,8 +261,8 @@ <h3 class="manage-account__switch-title">&nbsp;</h3>
                         class="manage-account__button--icon manage-account__button manage-account__button--@{skin.getOrElse("main")}"
                         data-link-name="consents : navigation : submit"
                     >
-                        <span>Done</span>
-                        @fragments.inlineSvg("tick-button", "icon")
+                        <span>All done!</span>
```

Comment:
is this too enthusiastic for a button? I'd drop the exclamation. Graun tone of voice and whatnot

Reply:
Yeah, I spoke to Celine about the text on the button. 'Done' had a stronger implication that the button will confirm the prior actions. Again, Celine said that the Guardian language always borders between informal and formal. 'All done!' doesn't imply as strongly that the button is essential to confirming the selections. It's more like "all done now, take me somewhere else".